testsuite/MDAnalysisTests/core/test_universe.py

```diff
@@ -272,10 +272,14 @@ def test_load_multiple_args(self):
         assert_equal(len(u.atoms), 3341, "Loading universe failed somehow")
         assert_equal(u.trajectory.n_frames, 2 * ref.trajectory.n_frames)
 
-    def test_pickle_raises_NotImplementedError(self):
+    def test_pickle(self):
```

Comment:
Would it be worth testing more than PSF/DCD here?

Reply:
I think the test here is just to make sure **Universe** pickling works. Since all the readers will be tested in *test_multiprocessing.py*, I am not sure it is worth testing more here again.

Reply:
My understanding here is that some of the readers are going to behave a little bit differently under the hood for pickling (e.g. GSD). My comment here was more "is it worth doing a parametrize and looping over various file types to make sure none of them fail to pickle".